fatal/type/impl/foreach.h

```diff
@@ -19,15 +19,15 @@ namespace impl_fe {
 template <typename, typename>
 struct f {
   template <typename... Args>
-  static void g(Args &&...) {}
+  static inline void g(Args &&...) {}
 };
 
 template <typename... T, std::size_t... Indexes>
 struct f<list<T...>, index_sequence<Indexes...>> {
   static_assert(sizeof...(T) == sizeof...(Indexes), "size mismatch");
 
   template <typename Visitor, typename... Args>
-  static void g(Visitor &&visitor, Args &&...args) {
+  static inline void g(Visitor &&visitor, Args &&...args) {
```

Comment:
To follow-up on this PR....

With this patch applied:

```
$ relevant_binary=#..., built with gcc and -O3, before the s/fe/f/ commit
$ nm -C "$relevant_binary" | grep -w fatal::impl_fe::fe | wc -l
0
$ nm -C "$relevant_binary" | grep -w fatal::foreach | wc -l
0
```

Which was the goal.